packages/@ourworldindata/types/src/dbTypes/ChartConfigs.ts

```diff
@@ -6,6 +6,7 @@ export interface DbInsertChartConfig {
     id: string
     patch: JsonString
     full: JsonString
+    fullMd5?: string
```

Comment:
Yes - I added this in the PR description to describe this part of the journey:

> This PR adds a hash of the full config in the chart_configs table. I started out using a SHA-1 and was happy that R2 supports those in addition to MD5, only to discover later on when I wrote the sync tool that the S3 API list operation has no support for hashes other than MD5. Since the main point of the hash is facilitate efficient comparison between set of configs in the DB and those in R2 I then rewrote the hash to be md5. The hash is stored in base64 encoding since this is also what is used and returned in most api calls (the only exception is the ETAG where a hex serialization in double quotes is used 🤷)